e2e/cypress/support/index.js

```diff
@@ -85,13 +85,18 @@ Cypress.on('test:after:run', (test, runnable) => {
     }
 });
 
-// Reset config
-before(() => {
-    cy.apiLogin('sysadmin');
-    cy.apiUpdateConfig();
-});
-
 // Add login cookies to whitelist to preserve it
+// Reset config before each test
 beforeEach(() => {
     Cypress.Cookies.preserveOnce('MMAUTHTOKEN', 'MMUSERID', 'MMCSRF');
+    cy.apiLogin('user-1');
```

Comment:
Added reset of user props for both user-1 and sysadmin since some tests changes props especially locale and fails other tests